frontend/packages/pipelines-plugin/src/components/pipelines/detail-page-tabs/pipeline-details/PipelineDetails.tsx

```diff
@@ -12,13 +12,12 @@ import { getPipelineTaskLinks } from '../utils';
 
 interface PipelineDetailsProps {
   obj: PipelineKind;
-  customData: RouteTemplate[];
+  customData: {
+    templateNames: RouteTemplate[];
+  };
```

Comment:
I know Query Prefix isn't used in this component, but it would seem we have a common type that should be written somewhere and then referenced so the data remains available and common during refactors.